.gitignore

```diff
@@ -23,3 +23,9 @@ yarn-error.log*
 
 # Include necessary vscode files
```

Comment:
This is fine if it really helps your development.  The call to `c:/utils/MinimalADB/adb.exe start-server` probably doesn't work everywhere, but again if it helps you out, then it is fine.